### PR TITLE
feat(tiller): make configmaps the default storage

### PR DESCRIPTION
**Pull request by technosophos**
This adds a Tiller CLI flag to override the default, and tests to
make sure that the default comes up as expected.

### CODE REVIEW

File: cmd/tiller/tiller_test.go [hunk 1]
```diff
@@ -44,4 +46,22 @@ func TestInit(t *testing.T) {
...
+	var expect driver.ConfigMaps
+	v := reflect.Indirect(reflect.ValueOf(env.Releases))
+	tt := v.Type()
+	f0 := tt.Field(0)
+	if !f0.Anonymous {
+		t.Fatalf("No embeded driver in %T", tt)
+	}
+	if f0.Type.ConvertibleTo(reflect.ValueOf(expect).Type()) {
+		t.Fatalf("Driver should be %T, got %T", expect, f0.Type.Name())
+	}
```
Comment (adamreese) [inline cmd/tiller/tiller_test.go]: Could we include a `driver.Name()`?

File: cmd/tiller/tiller.go [hunk 2]
```diff
@@ -40,6 +47,7 @@ var env = environment.New()
 
 var addr = ":44134"
 var probe = ":44135"
+var store = storageConfigMap
```
Comment (adamreese) [inline cmd/tiller/tiller.go]: nifty nitty, group vars in `var (...)` to align with OCD proper standards

File: cmd/tiller/tiller.go [hunk 3]
```diff
@@ -38,8 +45,11 @@ var rootServer = grpc.NewServer()
 // Any changes to env should be done before rootServer.Serve() is called.
 var env = environment.New()
 
-var addr = ":44134"
-var probe = ":44135"
+var (
+	addr  = ":44134"
+	probe = ":44135"
+	store = storageConfigMap
+)
```
Comment (adamreese) [inline cmd/tiller/tiller.go]: So neat and tidy!

Comment (technosophos) [inline cmd/tiller/tiller.go]: I'm so glad you noticed that.

**Pull request merged**